crates/feldera-types/src/config.rs

```diff
@@ -401,6 +401,37 @@ pub struct SyncConfig {
     /// If the checkpoint doesn't exist in object store, the pipeline will
     /// fail to initialize.
     pub start_from_checkpoint: Option<StartFromCheckpoint>,
+
+    /// The number of file transfers to run in parallel.
+    /// Default: 20
+    pub transfers: Option<u8>,
+
+    /// The number of checkers to run in parallel.
+    /// Default: 20
+    pub checkers: Option<u8>,
+
+    /// Set to skip post copy check of checksums, and only check the file sizes.
+    /// This can significantly improve the throughput.
+    /// Defualt: false
+    pub ignore_checksum: Option<bool>,
+
+    /// Number of streams to use for multi-thread downloads.
+    /// Default: 10
+    pub multi_thread_streams: Option<u8>,
+
+    /// Use multi-thread download for files above this size.
+    /// Default: 100M
+    pub multi_thread_cutoff: Option<String>,
```

Comment:
can you include the unit in the name (e.g., `multi_thread_cutoff_bytes`). Please document the format of this string.

Reply:
Users can specify the unit. So it can be: 100M, or 1G.